### ISSUE TITLE
Include native names in language list in Settings

### ISSUE THREAD

**Issue by minna-xD** [TorstenDittmann/OmniaWrite#134]
Currently the language list in settings is translated in each of the target languages. I started to think that this probably doesn't serve the user best. Imagine a situation where the user accidentally clicks on a wrong language and doesn't understand the one he chose.

The clearest (and least effort) would perhaps be a list with the languages only in their native name (e.g. English, Deutsch, Suomi, Svenska, Nederlands...). In this case the language list would be identical in all language files:
```
      "language": {
        "title": "Language",
        "en": "English",
        "de": "Deutsch",
        "es": "Español",
        "fr": "Français",
        "fi": "Suomi",
        "it": "Italiano",
        "nl": "Nederlands",
        "pt": "Português",
        "ru": "Русский",
        "tr": "Türkçe",
        "ja": "日本語",
        "sv": "Svenska",
        "ur": "اردو"
      }
```

An alternative solution would be to have both the native name and the name in the currently selected language, which is e.g. the way LinkedIn shows the language list.

**Comment by TorstenDittmann** [comment]
Good call, will think of a neat way on how to implement something like this 👍 

Thanks :bow:
